ui/src/app/pages/homepage/recent-resources.tsx

```diff
@@ -29,16 +29,17 @@ export const RecentResources = fp.flow(withCdrVersions())((props: Props) => {
   const [apiLoadError, setApiLoadError] = useState<string>(null);
 
   const loadResources = async () => {
-    setLoading(true);
-    await userMetricsApi()
-      .getUserRecentResources()
-      .then(setResources)
-      .catch(() => {
-        setApiLoadError(
-          'An error occurred while loading recent resources. Please refresh the page to reload.'
-        );
-      })
-      .finally(() => setLoading(false));
+    try {
```

Comment:
why change this?

Reply:
The pop: Do you want to go to copied workspace, was broken on recent-resources as well. This change fixed it